sdk/kubeflow/trainer/api/trainer_client.py

```diff
@@ -513,10 +513,10 @@ def __get_trainjob_from_crd(
         # Add the TrainJob status.
         # TODO (andreyvelich): Discuss how we should show TrainJob status to SDK users.
         if trainjob_crd.status and trainjob_crd.status.conditions:
+            # The TrainJob exists at that stage so its status can safely default to Created
+            status = constants.TRAINJOB_CREATED
```

Comment:
Shall we assign the status before `trainjob_crd.status.condition` is added ?
E.g. `trainjob_crd.status.condition is None` until we add Complete or Failed status to TrainJob

Reply:
You're right! Moved outside the `if` block.